.travis.yml

```diff
@@ -5,6 +5,9 @@ rust:
   - nightly
 notifications:
   email: false
+script:
+  - cargo build --verbose
```

Comment:
I don't think we need these lines - integration tests only run if you specify "features aws_integration" or a service name.  By default they do not run.